src/sock_ctrl_msg.rs

```diff
@@ -0,0 +1,461 @@
+// Copyright 2017 The Chromium OS Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+//! Used to send and receive messages with file descriptors on sockets that accept control messages
```

Comment:
nit: Wrappers for sending and receiving ....

We should use nouns for what the module is.